src/CrowdinApiClient/Api/BranchApi.php

```diff
@@ -16,8 +16,8 @@ class BranchApi extends AbstractApi
 {
     /**
      * List Branches
-     * @link https://support.crowdin.com/api/v2/#operation/api.projects.branches.getMany API Documentation
-     * @link https://support.crowdin.com/enterprise/api/#operation/api.projects.branches.getMany API Documentation Enterprise
+     * @link https://developer.crowdin.com/api/v2/#operation/api.projects.branches.getMany API Documentation
+     * @link https://developer.crowdin.com/enterprise/api/#operation/api.projects.branches.getMany API Documentation Enterprise
```

Comment:
This link is broken

For Crowdin Enterprise it should be like this:

`https://support.crowdin.com/enterprise/api/*` -> `https://developer.crowdin.com/enterprise/api/v2/*`